### PR TITLE
added amq scale test

### PR DESCRIPTION
**Pull request by SvetlanaRedHat**
Signed-off-by: savetisy <savetisy@redhat.com>

### CODE REVIEW

**Review by Avilir — COMMENTED** [review]
except some flax8 errors, looking good.

**Review by vasukulkarni — DISMISSED** [review]

File: tests/e2e/scale/test_scale_amq.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    amq = AMQ()
+
+    def teardown():
+        amq.cleanup()
+
+    request.addfinalizer(teardown)
+    return amq
+
+
+@performance
```
Comment (keesturam) [inline tests/e2e/scale/test_scale_amq.py:25]: shouldn't this be scale?

**Review by Akarsha-rai — APPROVED** [review]
LGTM

File: tests/e2e/scale/test_scale_amq.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+class TestAMQBasics(E2ETest):
+    @pytest.mark.parametrize(
+        argnames=["interface"],
+        argvalues=[
+            pytest.param(
+                constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-1279")
+            )
+        ]
+    )
```
Comment (ramkiperiy) [inline tests/e2e/scale/test_scale_amq.py:34]: Are we going to use this test only for `constants.CEPHBLOCKPOOL` or we have any future scope? 
If no scope then better to remove these lines, add like below in line48
`sc = default_storage_class(interface_type=constants.CEPHBLOCKPOOL)`, also you can add the polarion_id marker something like `@pytest.mark.polarion_id("OCS-2321")` in line 25/26

Comment (SvetlanaRedHat) [reply tests/e2e/scale/test_scale_amq.py:34]: added polarion_id, for now its wil be in scope of constants.CEPHBLOCKPOOL

Comment (SvetlanaRedHat) [reply tests/e2e/scale/test_scale_amq.py]: it will be for  ceph for now,  I will change and extend in the future if needed
fixed polarian_id

**Review by vasukulkarni — APPROVED** [review]

**Review by ramkiperiy — APPROVED** [review]

**Review by wusui — APPROVED** [review]
Looks okay

**Pull request merged**